frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java

```diff
@@ -36,28 +35,28 @@
   private static final String TOGGLE_LIST_COMMIT_TEXT = "Toggle List Commits";
   private static final String TOGGLE_LIST_COMMIT_DESCRIPTION = "Toggle list commits";
 
-  private final GitMacheteGraphTableManager gitMacheteGraphTableManager;
+  private final IGraphTableManager gitMacheteGraphTableManager;
 
   @UIEffect
   public GitMachetePanel(Project project) {
     super(/* vertical */ false, /* borderless */ true);
-    LOG.debug("Instantiation of GitMachetePanel");
+    LOG.debug("Instantiation of ${getClass().getSimpleName()}");
```

Comment:
Wrt. #214 that has been merged something like `LOG.debug("Instantiation")` is enough

Reply:
Fixed on PR #233